.gitignore

```diff
@@ -1,4 +1,5 @@
 bin/*
 node_modules/*
-
 todomvc/*
+examples/*.js
```

Comment:
if we do not exclude the generated files, we can use RawGit to display the demos online.